terraform/modules/website-e2e-test/testing.tf

```
@@ -8,7 +8,9 @@ resource "google_cloudbuild_trigger" "testing_web_e2e_run_tests_trigger" {
   included_files = [
     "website/**",
     "ops/web-e2e.cloudbuild.yaml",
-    "terraform/modules/website-e2e-test/**"
+    "terraform/modules/website-e2e-test/**",
+    "testing/website-e2e/**",
+    "ops/e2e-runner/**"
```

Revisiting: we don't build the Dockerfile as part of tests currently so making this change won't help right now.
```suggestion
    "testing/website-e2e/**"
```